app/models/service_specific_post.rb

```diff
@@ -0,0 +1,3 @@
+class ServiceSpecificPost < ApplicationRecord
+  validates :title, presence: true
+end
```

Comment:
I like it - keeps the data manipulation in the model. FYI, using default_scope like that works fine.